data/json/professions.json

```
@@ -1186,6 +1186,16 @@
       "female": [ "bra", "boy_shorts" ]
     }
   },
+  {
+    "type": "profession",
+    "id": "heli_pilot_scenario",
+    "name": "Helicopter Pilot",
+    "description": "You earned a living ferrying businessmen and tourists from helipad to helipad, now your flying skills are all you have to rely on.",
```

```suggestion
    "description": "You earned a living ferrying businesspeople and tourists from helipad to helipad, now your flying skills are all you have to rely on.",
```
just a tiny nitpick.